src/Root.cpp

```diff
@@ -252,7 +252,7 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 
 		LOG("Shutting down server...");
 		m_Server->Shutdown();
-	}  // if (m_Server->Start()
```

Comment:
Why was this comment removed?

Reply:
It didn't seem particularly useful to me :P

Reply:
I think it's probably better left - it just gives an indication of what the long indented block was for.